### DISCUSSION TITLE
Welcome to Data Science Jumpstart with 10 Projects Discussions!

### DISCUSSION THREAD

**Discussion by mikeckennedy** [talkpython/data-science-jumpstart-with-10-projects-course#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kalifg** [comment]
Hi!  My name is Michael Dwyer and I am taking this course in order to become more familiar with Python data science tools.

In one part of the second notebook (02-income), there is a code section that calls `import helpers` so that it can use `helpers.jitter`.

I'm opening the notebook in PyCharm and it tells me that this library is not installed.  It is not in the `requirements.txt`. 

I did try installing `helpers` manually but it does not seem to be the one that is needed; it does not export a `jitter` function.

Could you please point me in the right direction?

Thank you!

**Reply by mattharrison** [reply]
Hi @kalifg 

Thanks for taking the course. I uploaded the `helpers.py` file in the notebook directory. 

Best!

**Reply by kalifg** [reply]
Thank you!

**Comment by kalifg** [comment]
Note for Project 9: SQL Integration
My dates were coming out of the database in the following slightly different format: `1990-01-01T00:00:00-07:00` (note the `"T"`).  After I altered the format strings to account for it everything worked as expected.

**Comment by mikeckennedy** [comment]
Thanks for the extra info @kalifg. Did you figure out why the slight difference in format? What's your locale (de, en-us, en-uk, etc)? And @mattharrison thanks for adding the file. :)

**Reply by kalifg** [reply]
My locale should be en-us, but I'm not sure how to check it in this case.  I'm using PyCharm's built-in Jupyter notebook viewing in case that makes a difference.

**Comment by mikeckennedy** [comment]
@kalifg Got it. Possibly PyCharm is passing En rather than en-us for the time? I don't know. Thanks for the extra info.
